docs/integrating.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: integrating
+append_help_link: true
+description: >-
+  Learn about integrating Semgrep and the Semgrep Registry into your product.
+---
+
+# Semgrep integration guide for partners
+
+
+### Introduction
+
+We're excited that you're integrating Semgrep into your tooling! Our goal with Semgrep is to bring world-class security tools to developers based on our conviction that software will run the most exciting parts of the future. It's not something that we can do alone; we want to build a community around sharing programmatic knowledge about how to build secure software.
+
+
+### Requirements for Integrators
```

Comment:
```suggestion
### Requirements for integrators
```
[Use sentence case!](https://developers.google.com/style/capitalization#:~:text=In%20document%20titles%20and%20headings,always%20capitalized%20a%20certain%20way.)